spec/features/workflow/withdraw_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+RSpec.feature "Withdraw a document" do
+  scenario do
+    given_there_is_a_published_edition
+    and_i_have_the_managing_editor_permission
+    when_i_visit_the_summary_page
+    and_i_click_on_withdraw
+    then_i_see_that_i_can_withdraw_the_document
+    when_i_fill_in_the_public_explanation
+    and_click_on_withdraw_document
+    then_i_see_the_document_has_been_withdrawn
+
+    when_i_click_to_change_the_public_explanation
+    then_i_can_see_the_existing_public_explanation
+    and_i_can_edit_the_public_explanation
```

Comment:
I'd suggest that editing a withdrawal is a separate feature that we'd want to test rather than grouping them.